phobos_umss.sh

```diff
@@ -0,0 +1,600 @@
+#!/bin/bash
+
+# All rights reserved (c) 2014-2024 CEA/DAM.
+#
+# This file is part of Phobos.
+#
+# Phobos is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 2.1 of the License, or
+# (at your option) any later version.
+
+# Phobos is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Phobos. If not, see <http://www.gnu.org/licenses/>.
+#
+# \brief  Phobos UMSS script for iRODS
+
+
+# This script is the Phobos implementation of an iRODS Universal Mass Storage
+# System plugin (UMSS), enabling the use of Phobos as a storage tier by iRODS.
+#
+# In order to be detected by iRODS, the present file is to be placed in
+# /var/lib/irods/msiExecCmd_bin/
+
+VERSION=v1.2
+ID=$RANDOM
+STATUS="FINE"
+LEVEL=1
+DEBUG=1
+DISPLAY=0
+DISPLAY_COLOUR=1
+DISPLAY_BLACK=0
+
+LOGFILE_FANCY=/var/lib/irods/$VERSION.log
+#LOGFILE_SOBRE=/var/lib/irods/$VERSION.txt
+sudo touch $LOGFILE_FANCY
+#sudo touch $LOGFILE_SOBRE
+sudo chown "$(whoami)":"$(whoami)" $LOGFILE_FANCY
+#sudo chown "$(whoami)":"$(whoami)" $LOGFILE_SOBRE
```

Comment:
It happens that the default location for iRODS is in `/var/lib`. So to create the log file, we need sudo privileges. From this point, we could write in this file requiring sudo privileges all the time, but it would be strange having one file belonging to `root` among a directory belonging to `irods` user. So we make this file belonging to `irods`.